client/pages/tournaments.jsx

```diff
@@ -79,11 +111,24 @@ export default class Tournaments extends React.Component {
     if (route.path === 'tournaments') {
       return (
         <>
-          <TourneySlider />
-          <SubHeader text="Open Tournaments" />
-          <Cards
-            tourneys={this.state.tourneys}
-            numParticipants={this.state.numParticipants} />
+          <TourneySlider sliderSelected={this.state.slider} onSliderClick={this.handleSliderClick}/>
+          <SubHeader text={this.state.headerText} />
+          <div className="cards-container">
+            {
+              this.state.tourneys.map(tourney => {
+                if (!tourney.closed) {
+                  return (
+                    <Card key={'card-' + tourney.tourneyId}
+                      id={tourney.tourneyId}
+                      tourney={tourney}
+                      src="./images/hero-banner.jpg" />
+                  );
+                } else {
+                  return <></>;
```

Comment:
This if fine but also you can return `null` if you don't want anything to show up, just FYI.